### PR TITLE
GroupDreamTeamTest| Refactoring class

### PR DESCRIPTION
**Pull request by darinamas**

### CODE REVIEW

File: src/test/java/NoGroupTest.java [hunk 1]
```diff
@@ -13,6 +14,7 @@
 
 public class NoGroupTest extends BaseTest {
 
+    @Ignore
```
Comment (darinamas) [inline src/test/java/NoGroupTest.java:17]: @Rom-T Tested failed please fix.

Comment (Rom-T) [reply src/test/java/NoGroupTest.java:17]: Ok. Thank you.

File: src/test/java/JkTest.java [hunk 2]
```diff
@@ -2,11 +2,13 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.testng.Assert;
+import org.testng.annotations.Ignore;
 import org.testng.annotations.Test;
 import runner.BaseTest;
 
 
 public class JkTest extends BaseTest {
+        @Ignore
```
Comment (darinamas) [inline src/test/java/JkTest.java:11]: @julykan Test failed, please fix.

File: src/test/java/GroupCleanCodeTest.java [hunk 3]
```diff
@@ -19,7 +20,7 @@ public void testFolkInstruments() {
 
         Assert.assertEquals(link.getText(), "ritual scepter");
     }
-
+    @Ignore
```
Comment (darinamas) [inline src/test/java/GroupCleanCodeTest.java:23]: @Vorsoba Test failed, please fix.

**Review by InaChemerko — APPROVED** [review]
LGTM

**Review by Elgrueber — APPROVED** [review]
LGTM

**Pull request merged**